docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3"
+services:
+  web:  # TODO this is using Django's built in development server,
+        # need to figure out setting up an actual server using Apache
+    build: .
+    container_name: map_backend
+    command: python3 backend/manage.py runserver 0.0.0.0:8000
+    volumes:
+        - .:/backend
+    ports:
+        - "8000:8000"
+    env_file:
+        - ./.env.dev
+    depends_on:
+        - postgres
+
+  postgres:
```

Comment:
I think we'll need to move postgres to take place before the "web" service. The first time docker-compose is run, the web service will fail to start because the DB isn't set up yet.